src/JsonService.ts

```diff
@@ -27,6 +32,35 @@ export interface PostFormOpts {
     initCredentials?: "same-origin" | "include" | "omit";
 }
 
+/**
+ * @internal
+ */
+function appendCustomHeaders(
```

Comment:
please make this an private function within JsonService class

Reply:
I've made the modification in my last commit.